Mathlib/Analysis/Convex/Star.lean

```diff
@@ -68,7 +67,9 @@ variable (𝕜) [SMul 𝕜 E] [SMul 𝕜 F] (x : E) (s : Set E)
 
 /-- Star-convexity of sets. `s` is star-convex at `x` if every segment from `x` to a point in `s` is
 contained in `s`. -/
-def StarConvex : Prop :=
+@[nolint unusedArguments]
+def StarConvex (𝕜 : Type*) {E : Type*} [Semiring 𝕜] [PartialOrder 𝕜] [IsOrderedRing 𝕜]
```

Comment:
Again I would be tempted to drop `IsOrderedRing` in this definition and then pick up the pieces rather than using `nolint`. You can add a `TODO` or whatever saying "this file probably needs reordering because we have 30 `omit`s", and leave it for another PR, but basically this is the system telling us that the file needs reordering.

If you really do not want to do this, then at least you should put a `TODO` in the module doc saying that `IsOrderedRing` should be dropped from the def of `StarConvex`. But adding `nolint` is adding technical debt and we would ideally avoid this; I think a mountain of `omit`s is better.